charts/internal/seed-controlplane/charts/csi-driver-controller/templates/deployment-csi-driver-controller.yaml

```diff
@@ -117,15 +116,14 @@ spec:
         - --kubeconfig=/var/run/secrets/gardener.cloud/shoot/generic-kubeconfig/kubeconfig
         - --leader-election
         - --leader-election-namespace=kube-system
-        - --timeout={{ .Values.timeout }}
```

Comment:
Sorry my bad for this. But I guess with this generic loop it can happen that we don't have any flag for verbosity or timeout nor default values for them. I guess we should only make `extraArgs` configurable that we know that they can be configured via the `cloudprofile`.

same for the other csi components. wdyt?

Reply:
> But I guess with this generic loop it can happen that we don't have any flag for verbosity or timeout nor default values for them.

That is true. It would be on the user to decide if he wants to differ from the csi component default not on gardener anymore.

> I guess we should only make extraArgs configurable that we know that they can be configured via the cloudprofile.

Can't really follow. Isn't this what I proposed with the map[string]string?